### PR TITLE
Initial commit

### LINKED ISSUE #1053: Add tests to test Transactional and FT
Add tests to test Transactional and FT in the FT test bucket.

### PR DESCRIPTION
**Pull request by jonhawkes**
fixes #1053 

#build

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/.classpath [hunk 1]
```diff
@@ -1,9 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/TxFaultTolerance.war/src"/>
 	<classpathentry kind="src" path="test-applications/CDIFaultTolerance.war/src"/>
 	<classpathentry kind="src" path="test-applications/faulttolerance.jar/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/com.ibm.websphere.javaee.cdi.2.0/generated/com.ibm.websphere.javaee.cdi.2.0.jar">
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/.classpath]: This isn't right ... should pull the api stuff in via the bnd container. Only the test app src folder needs adding here.

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/tests/TxRetryTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/tests/TxRetryTest.java]: wrong date

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/bootstrap.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/bootstrap.properties]: wrong date

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/server.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/server.xml]: wrong date

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/server.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+<server>
+
+    <include location="../fatTestPorts.xml"/>
+    
+	<featureManager>
+	    <feature>componenttest-1.0</feature>
+		<feature>osgiconsole-1.0</feature>
+		<feature>localConnector-1.0</feature>
+		<feature>appSecurity-2.0</feature>
+		<feature>cdi-1.2</feature>
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultTolerance/server.xml]: Emily changed the FaultTolerance bucket to run against cdi-2.0 by default and then just selected tests with cdi-1.2. You might wish to follow the same pattern.

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/test-applications/TxFaultTolerance.war/resources/META-INF/microprofile-config.properties [hunk 6]
```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/test-applications/TxFaultTolerance.war/resources/META-INF/microprofile-config.properties]: ...not going to comment on every date...

Comment (jonhawkes) [reply dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/test-applications/TxFaultTolerance.war/resources/META-INF/microprofile-config.properties]: no. that would be silly.

**Pull request closed without merging**